### PR TITLE
Citybike Wien stopped operations 1st April, replaced by WienMobil Rad

### PR DESCRIPTION
**Pull request by Robot8A**

### CODE REVIEW

File: data/brands/amenity/bicycle_rental.json [hunk 1]
```diff
@@ -1060,16 +1060,17 @@
...
-        "network": "Citybike Wien",
-        "network:wikidata": "Q14541491",
-        "operator": "Citybike Wien"
+        "brand": "WienMobil Rad",
+        "brand:wikidata": "Q111794110",
+        "network": "WienMobil Rad",
+        "network:wikidata": "Q111794110",
+        "operator:wikidata": "Q2351279",
+        "operator": "Nextbike",
+        "operator:wikidata": "en:Nextbike"
```
Comment (LaoshuBaby) [inline data/brands/amenity/bicycle_rental.json:1073]:
```suggestion
```

Hey! looks like you are trying to add a wikipedia? After discussion in #6481 NSI finally stop suggest wikipedia content so this is not needed. Thank you~

File: data/brands/amenity/bicycle_rental.json [hunk 2]
```diff
@@ -1060,16 +1060,17 @@
       }
     },
     {
-      "displayName": "Citybike Wien",
-      "id": "citybikewien-cda769",
+      "displayName": "WienMobil Rad",
       "locationSet": {"include": ["at"]},
```
Comment (LaoshuBaby) [inline data/brands/amenity/bicycle_rental.json:1064]:
```suggestion
      "locationSet": {"include": ["at"]},
      "matchNames": ["Citybike Wien"],
```
If this is a name change modify you'd better add old name as a matchName.

**Review by LaoshuBaby — APPROVED** [review]

File: data/brands/amenity/bicycle_rental.json (hunk 2 above)
Comment (Robot8A) [reply data/brands/amenity/bicycle_rental.json:1064]: They are not the same bicycle sharing system. Rather, the first one (Citybike Wien) expired its contract with the city council, which selected a different company (Nextbike) to run a new bicycle sharing system (WienMobil Rad) with new stations and bicycles.

Comment (Robot8A) [reply data/brands/amenity/bicycle_rental.json:1064]: WienMobil Rad started operations on the 1st April 2022, just one day after Citybike Wien stopped operating.

**Pull request merged**